components/KebabPopUp.tsx

```diff
@@ -21,9 +21,10 @@ export const KebabPopUp: React.FC<KebabPopUpProps> = (props) => {
         type={props.iconType}
         color={Theme.Colors.GrayIcon}
       />
-      <BottomSheet
+      <Overlay
```

Comment:
You seem to have changed the UX here^, please keep screenshots handy and paste it in PR descriptions & Jira ticket.

^ I'm taking a guess here by referring to https://reactnativeelements.com/docs/2.3.2/bottomsheet

Reply:
I will attach the screen record for in PR descriptions & Jira ticket.